common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Image.scala

```diff
@@ -0,0 +1,23 @@
+package uk.ac.wellcome.models.work.internal
+
+sealed trait BaseImage[+Id] extends HasIdState[Id] {
```

Comment:
👍 Looks good to me! I take it that `MergedImage` will be the output of the merger?